src/component/sidebar/index.tsx

```diff
@@ -33,13 +34,25 @@ function Sidebar(props: SidebarProps) {
   );
   const uniqueTimelineVentureIds = [...new Set(ventureIds)];
 
-  const { data: venturesData } = useVentureByTimeline({
-    ventureIds: uniqueTimelineVentureIds,
-    token,
-  });
+  const { data: ventureByTimelineData = [], isSuccess: ventureSuccess } =
+    useVentureByTimeline({
+      ventureIds: uniqueTimelineVentureIds,
+      token,
+    });
+
+  const { data: ventureByUserData = [], isSuccess: ventureUserSuccess } =
```

Comment:
Fixes https://github.com/venturemark/webclient/issues/324

It also means one extra network request before the sidebar can be displayed. We should be able to get these from the context without loading them in this component. We can improve that later.